app/views/challenges/48in24.html.haml

```diff
@@ -1,11 +1,12 @@
 - week = ((Time.zone.today - Date.new(2024, 1, 15)) / 7).ceil
-- featured_exercise = @exercises.find { |e| e.week == week }
+- exercises = @exercises.select { |e| e.week <= week }
```

Comment:
I like the old naming a bit better

```suggestion
- live_exercises = @exercises.select { |e| e.week <= week }
```

Reply:
Happy to change it back. It had become messy in the interviening commits. @dem4ron  could you do that pls?